project/build.scala

```diff
@@ -24,8 +24,8 @@ object BuildSettings {
     scalacOptions in(Compile, doc) := Seq("-groups", "-implicits"),
     description := "RDF framework for Scala",
     startYear := Some(2012),
-    resolvers += Resolver.bintrayRepo("inthenow","releases")
-    
+    resolvers += Resolver.bintrayRepo("inthenow","releases"),
+    updateOptions := updateOptions.value.withCachedResolution(true) //to speed up dependency resolution
```

Comment:
I am tired of long loading time (esp. when I am asked to change one line in my PR and I have to open and wait until it loads=)) . I use cacheresolution in most of my projects and never had any troubles with it, but speed ups were pretty sufficient.